### PR TITLE
update status of commands

### PR DESCRIPTION
**Pull request by itsmurugappan**
## Description

Update status of Export/Import/Gitops to Beta

cc @rhuss @dsimansk

### CODE REVIEW

**Comment by dsimansk** [comment]
I haven't found `kn container` in the mix. That should be `Beta` as well.

File: pkg/kn/commands/domain/create.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func NewDomainMappingCreateCommand(p *commands.KnParams) *cobra.Command {
 		Use:   "create NAME",
 		Short: "Create a domain mapping",
 		Example: `
-  # Create a domain mappings 'hello.example.com' for Knative service 'hello' with TLS secret set
+  # Create a domain mappings 'hello.example.com' for Knative service 'hello' with TLS secret set (Beta)
```
Comment (rhuss) [inline pkg/kn/commands/domain/create.go:36]: I wonder whether we should mark the domain management as "Beta" or whether we say that it's already GA ? I think we can move it to GA level as the UI is straightforward and not really contentious.

**Review by rhuss — COMMENTED** [review]
Thanks, looks good to me, but I wonder whether we should still mark the domain management as 'beta' ?

File: pkg/kn/commands/domain/create.go (hunk 1 above)
Comment (dsimansk) [reply pkg/kn/commands/domain/create.go:36]: +1 to GA

**Comment by dsimansk** [comment]
@itsmurugappan thanks!

Looks good now, let's get it in.
/approve
/lgtm

**Pull request merged**